src/controllers/clients.js

```diff
@@ -0,0 +1,31 @@
+const dbConfig = require('../../knexfile');
+const environment = process.env.NODE_ENV || 'development';
+const knex = require('knex')(dbConfig[environment]);
+
+const errorRes = require('../utils/responses/errorResponse');
+const successRes = require('../utils/responses/successResponse');
+const validateClient = require('../utils/validators/validateClient');
+
+const registerClient = async (req, res) => {
+  const { nome: name, email, cpf } = req.body;
+
+  try {
+    const messageError = await validateClient(knex, { name, email, cpf });
+    if (messageError) return errorRes.errorResponse404(res, messageError);
```

Comment:
404 é erro not Found colocamos esse em todas as respostas da primeira sprint mas como é erro dos dados enviados para a api o correto é o erro 400 temos que corrigir